docs/topics/http/middleware.txt

```diff
@@ -312,7 +312,7 @@ If your middleware has both ``sync_capable = True`` and
 ``async_capable = True``, then Django will pass it the request without
 converting it. In this case, you can work out if your middleware will receive
 async requests by checking if the ``get_response`` object you are passed is a
-coroutine function, using :py:func:`asyncio.iscoroutinefunction`.
+coroutine function, using ``asyncio.iscoroutinefunction``.
```

Comment:
Please undo. The previous form linked to the asyncio documentation. Also this file is unrelated to your change.

Reply:
Ah, I guess you changed this due to a failing reference with the Python 3.11 docs? This was fixed in #16234.